lms/envs/aws.py

```diff
@@ -1099,6 +1099,18 @@
 INSTALLED_APPS.append('openedx.features.ucsd_features')
 
 ############### Settings for Caliper Tracking #####################
+# 'caliper_tracking' app enables us to transform Edx event-logs according to
+# IMSGlobal Caliper Standards.
+# https://www.imsglobal.org/sites/default/files/caliper/v1p1/caliper-spec-v1p1/caliper-spec-v1p1.html.
+#
+# Dependencies:
+#   1. This app adds its processor in both the event tracking engine's settings
```

Comment:
Please make this more descriptive. Explain exactly what setting it changes in `EVENT_TRACKING_BACKEND` AND `TRACKING_BACKENDS` and what is the difference between them.